travis/travis_weblate.py

```diff
@@ -204,22 +204,26 @@ def update(self):
         with self.wl_api.componet_lock():
             self._git.run(["fetch", "origin"])
             first_commit = False
+            component = [item for item in self.wl_api.components
```

Comment:
That change is required because the weblate create only one repository for all components